backend/infrahub/core/attribute.py

```diff
@@ -607,6 +628,12 @@ class String(BaseAttribute):
     type = str
     value: str
 
+    @classmethod
+    def validate_content(cls, value: Any, name: str, schema: AttributeSchema) -> None:
+        super().validate_content(value, name, schema)
+        if ATTRIBUTE_TYPES[schema.kind] in [Text, Password]:
```

Comment:
I'd add `URL` here as well, don't think it will ever be relevant but you never know..

Reply:
I missed this one, I don't think we should hardcode the list if kinds that will be validated here, mostlikely this list will grow in the future and we'll forget about it
Might be better to have a list of kinds that are exempted from the check

Reply:
Looking closer into into it, ideally we should apply the validation to all attributes, after `serialize`, (except if the kind is exempted from the check)

Reply:
As discussed, added modifications to validate any non-large attribute. This is done after serializing to avoid an extra serialization.